MAUI/Autocomplete/Selection.md

```diff
@@ -141,6 +141,56 @@ The selected items can be changed programmatically by using the [SelectedItems](
 
 {% endtabs %}
 
+### TokensWrapMode
+
+There are two ways to display multi selection items in the AutoComplete control. They are
```

Comment:
There are two ways to display multi-selection items in the AutoComplete control. They are: